### DISCUSSION TITLE
USB Sound Card

### DISCUSSION THREAD

**Discussion by djiesr** [mcguirepr89/BirdNET-Pi#1107] [General]
I try three different UYB sound card with my Raspberry pi 3b.
1. A cheap Aliexpress audio adapter : [Aliexpress](https://s.click.aliexpress.com/e/_DFKiTbT) $5 US. The record get a lot of noise, but I detect a some bird close of the microphone.
2. Sabrent USB External Sound Adapter AU-MMSA on Amazon:  [Amazon](https://s.click.aliexpress.com/e/_DFKiTbT) I got a very compress audio record, I got less noise of the Aliexpress, but a bad audio quality.
3. UGREEN USB adapter: [Amazon ](https://amzn.to/3O8tkDh)Finally, I got the UGREEN audio adaptor and the record get less noise and a better quality of the two other one. 
So, it's just my experience, but I got a Big difference with the UGREEN device. I hope that will help you.

**Comment by grychofr** [comment]
I agree with you Ugreen is very good, better than Powerdewise I also tried and then ask for a refund.

But you do not need a 2way soundcard : these models from Ugreen are enough : ASIN B08Y8CZB2S or without cable : B087T5H3MQ and less expensive.

I use both (I have 5 stations in France) with and without cable. I see no difference

**Comment by frankal** [comment]
@djiesr @grychofr thank you for this advice.
Can I ask
1) what mic do you use? and
2) the ugreen sound cards are TRRS connection - do you use a TRRS cable? or an adaptor (TRS > TRRS)?

**Reply by grychofr** [reply]
I use AGPTEK mic (Amazon ASIN : B07SHSHW6H). It has TRRS plug and it is not expensive (12 euros).

**Comment by stevedee** [comment]
This may be of interest:-

http://captainbodgit.blogspot.com/2024/12/birdnet-pi-usb-audio-module-testing.html
